### PR TITLE
Adds support for debian/bookworm

### PR DESCRIPTION
**Pull request by gurkanindibay**
Ubuntu/kinetic packagecloud support is not ready yet. After packagecloud support for ubuntu/kinetic, the packagecloud code should be added

### CODE REVIEW

File: .github/workflows/build-package-test.yml [hunk 1]
```diff
@@ -74,7 +75,7 @@ jobs:
           POSTGRES_VERSION: ${{ matrix.POSTGRES_VERSION }}
 
       - name: Clone tools repo for test
-        run: git clone -b v0.8.22 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b ubuntu_kinetic_support --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (gurkanindibay) [inline .github/workflows/build-package-test.yml]: will be changed after merge of PR https://github.com/citusdata/tools/pull/263

File: .github/workflows/build-package.yml [hunk 2]
```diff
@@ -76,7 +77,7 @@ jobs:
           POSTGRES_VERSION: ${{ matrix.POSTGRES_VERSION }}
 
       - name: Clone tools repo for test
-        run: git clone -b v0.8.22 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b ubuntu_kinetic_support --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/build-package.yml]: Don't forget to update this line before merge.

File: .github/workflows/build-package-test.yml (hunk 1 above)
Comment (hanefi) [reply .github/workflows/build-package-test.yml]: Don't forget to update this line before merge.

**Review by hanefi — APPROVED** [review]
Needs 2 updates before merge.

**Pull request merged**